app/facades/header_facade.rb

```diff
@@ -0,0 +1,17 @@
+class HeaderFacade
+  def merchants
+    service.merchants[:data].map {|merchant| Merchants.new(merchant)}
+  end
+
+  def one_merchant(id)
+    OneMerchant.new(service.one_merchant(id)[:data])
```

Comment:
I think this can be `Merchants.new(service.one_merchant(id)[:data])` and then you can get rid of the `OneMerchant` poro altogether. The two classes appear to be identical.